src/lib/docker-utils.ts

```diff
@@ -367,13 +367,27 @@ export async function isV2DeltaImage(imageName: string): Promise<boolean> {
 	return inspect.Size < 40 && inspect.VirtualSize < 40;
 }
 
-const getAuthToken = memoizee(
-	async (
+const getAuthToken = (() => {
+	const memoizedGetToken = memoizee(
```

Comment:
I separated this out so as to easily include only the values actually relevant to the request within the memoization (and ignore any extra info that may be passed in but is unnecessary)